openmc/mesh.py

```diff
@@ -961,6 +961,44 @@ def build_cells(self, bc=None):
 
         return root_cell, cells
 
+
+    def get_index_where(self, value: float, basis: str = 'xy'):
```

Comment:
```suggestion
    def get_slice_index_where(self, value: float, basis: str = 'xy'):
```
perhaps this is a better method name?